### PR TITLE
Password UI improvements

### PR DESCRIPTION
**Pull request by bitboxer**
closes [#122](https://github.com/railslove/recover-backlog/issues/122)

### CODE REVIEW

File: lib/validators/passwordValidator.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as Yup from 'yup'
+
+export const passwordValidator = Yup.string()
+  .required('Passwort muss angegeben werden.')
+  .max(128, 'Das Passwort darf nicht länger als 128 Zeichen sein.')
```
Comment (bitboxer) [inline lib/validators/passwordValidator.ts:5]: Devise in the backend has a max length for passwords

Comment (bonflintstone) [reply lib/validators/passwordValidator.ts:5]: I feel like this gh comment would be more useful as a ts comment :grin:

File: lib/validators/passwordValidator.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  .matches(
+    /^(?=.*[A-Za-z])(?=.*\d)(?=.*[$&+,:;=?@#|'<>.^*()%!-[\]{}])[A-Za-z\d$&+,:;=?@#|'<>.^*()%!-[\]{}]{8,128}$/,
```
Comment (bonflintstone) [inline lib/validators/passwordValidator.ts:7]: Please tell me you did not come up with this regex yourself :smile:

Comment (bonflintstone) [reply lib/validators/passwordValidator.ts:7]: Oh I see you only moved it, and did not introduce it. I wonder where it comes from though

**Review by bonflintstone — APPROVED** [review]

File: lib/validators/passwordValidator.ts (hunk 2 above)
Comment (bitboxer) [reply lib/validators/passwordValidator.ts:7]: I bet it's Stackoverflow 🙈

**Pull request merged**